prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml

```diff
@@ -0,0 +1,155 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+  FHS: '/FHS'
```

Comment:
does this exist in our mappings? If so we should use this and not the "receiving application" settings in the receiver.yml

Reply:
It wasn't working which is why I switched it to the receiving application settings., Will remove the constant